doc/input-output-reference/src/overview/group-setpoint-managers.tex

```diff
@@ -195,7 +195,18 @@ \subsubsection{Inputs}\label{inputs-2-037}
 
 \paragraph{Field: Control Variable}\label{field-control-variable-2}
 
-The type of variable that will be controlled. There is only one choice for this type of setpoint manager: Temperature.
+The type of variable that will be controlled. There are three key choices for this type of set point manager:
+
+\begin{itemize}
+\item
+  Temperature
+\item
+  MaximumTemperature
+\item
+  MinimumTemperature
+\end{itemize}
+
+If the Control Variable input field is set to MaximumTemperature, then the setpoint will be set to a setpoint value specified at Setpoint at Outdoor Low Temperature input field, else if the Control Variable input field is set to MinimumTemperature, then the setpoint will be set to a setpoint value specified in Setpoint at Outdoor High Temperature input field, else if the Control Variable input field is set to Temperature, then the setpoint will be set to a setpoint value determined based on the current time outdoor air temperature, and the setpoint values specified at Setpoint Outdoor Low and High Temperature input fields.
```

Comment:
Following the pattern of SetpointManager:FollowOutdoorAirTemperature, this description can simply be:
Field:  Control Variable
The type of variable that will be controlled. There are three choices available: Temperature, MaximumTemperature, or MinimumTemperature.  

Then to clarify things in general for dual setpoint plantloops.  In the description for PlantLoop, Field: Plant Loop Demand Calculation Scheme, replace the first paragraph with the following.  (Please review this carefully and edit as needed.)

There are two choices for plant loop demand calculation scheme:

SingleSetpoint  -  For the SingleSetpoint scheme, the plant loop is controlled to a single temperature setpoint. This requires a setpoint manager with Control Variable = Temperature to place a setpoint on the Loop Temperature Setpoint Node.  The setpoint will be stored in the Node.TempSetPoint variable which may be reported using Output:Variable "System Node Setpoint Temperature." Examples of applicable setpoint managers include: SetpointManager:Scheduled, SetpointManager:OutdoorAirReset, SetpointManager:FollowOutdoorAirTemperature, etc. SingleSetpoint is the default.

DualSetpoint  -  For the DualSetpoint scheme, the plant loop is controlled to stay between a pair of high and low temperature setpoints. This requires one or two setpoint managers to place two setpoints on the Loop Temperature Setpoint Node.  The setpoints will be stored in the Node.TempSetPointHi and Node.TempSetPointLo  variables which may be reported using Output:Variable "System Node Setpoint High Temperature" and "System Node Setpoint Low Temperature." There is special setpoint manager, SetpointManager:Scheduled:DualSetpoint, which will place both the high and low setpoints with Control Variable = Temperature.  Otherwise, two setpoint managers are required, one with Control Variable = MaximumTemperature and another with Control Variable = MinimumTemperature. The two setpoint managers may be the same kind or different, but care should be taken so that the MinimumTemperature is not set higher than the MaximumTemperature. Examples of applicable setpoint managers include: SetpointManager:Scheduled, SetpointManager:OutdoorAirReset, SetpointManager:FollowOutdoorAirTemperature, etc.